main/main.cpp

```diff
@@ -2069,10 +2073,14 @@ bool Main::iteration() {
 
 	bool exit = false;
 
-	Engine::get_singleton()->_in_physics = true;
-
 	for (int iters = 0; iters < advance.physics_steps; ++iters) {
 
+		if (agile_input_event_flushing) {
```

Comment:
Should `InputDefault::get_singleton()->is_using_input_buffering()` be used here too to avoid possible side effects on platforms other than Android, since the documentation says it's implemented only on Android?

Reply:
As far as my brain gets, it shouldn't be needed, because the buffer would just be empty. However, I think it's a good idea in any case. Let's be defensive. Pushing change in 3, 2, 1...